pkg/metrics/metrics.go

```diff
@@ -787,6 +791,16 @@ func CreateConfiguration(metricsEnabled []string) (Configuration, []prometheus.C
 			collectors = append(collectors, EventTSK8s)
 			c.EventTSK8sEnabled = true
 
+			EventLagK8s = prometheus.NewGauge(prometheus.GaugeOpts{
+				Namespace:   Namespace,
+				Name:        "event_lag_seconds",
+				Help:        "Lag (computed value) for Kubernetes events",
+				ConstLabels: prometheus.Labels{"source": LabelEventSourceK8s},
+			})
```

Comment:
Let's update this description per my other comment as soon as this passes CI.

Reply:
(Actually we might want to rename the variable if we update the description to be specific to CNI ADD -> pod cache lag time.)